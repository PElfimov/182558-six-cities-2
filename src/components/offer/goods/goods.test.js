import React from 'react';
import renderer from 'react-test-renderer';
import Goods from './goods';

const goods = [
  `Washer`,
  `Baby seat`,
  `Fridge`];
it(`HotelCard correctly renders after relaunch`, () => {
  const tree = renderer
    .create(
        <Goods
          goods={goods}
        />).toJSON();
  expect(tree).toMatchSnapshot();
});
