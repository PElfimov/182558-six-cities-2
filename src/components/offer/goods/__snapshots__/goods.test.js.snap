// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HotelCard correctly renders after relaunch 1`] = `
<div
  className="property__inside"
>
  <h2
    className="property__inside-title"
  >
    What's inside
  </h2>
  <ul
    className="property__inside-list"
  >
    <li
      className="property__inside-item"
    >
      Washer
    </li>
    <li
      className="property__inside-item"
    >
      Baby seat
    </li>
    <li
      className="property__inside-item"
    >
      Fridge
    </li>
  </ul>
</div>
`;
